app/javascript/components/ingredients/ingredients-table.vue

```diff
@@ -135,6 +135,13 @@ export default {
   props: {
     ingredients: { type: Array, required: true },
   },
+
+  data() {
+    return {
+      originalIngredients: [...this.ingredients].map(element => JSON.parse(JSON.stringify(element))),
```

Comment:
acá creo que basta con hacer `this.ingredients` o no? no es necesario desestructurarlo creo

Reply:
Cacha que esto lo intenté así a la primera, pero como hay un v-model por ahí con los ingredientes, se me actualizaba en ambos lados, entonces necesitaba el valor y no la referencia, así que tuve que hacer la copia de la lista, pero tmbn la copia de cada objeto para que no se actualizara en ambos.